### PR TITLE
glue only mutes if drank

### LINKED ISSUE #16753: glue should only work when eaten
## Description
injecting it shouldnt be mute toxin

see glue spear, 20 seconds mute per hit

**Reproduction**
1. do glue like a heroin addict
2. cant speak because ?????

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
fixes #16753 no more glue spears and arrows

also makes glue fix bleeding on touch :trollface:

## Why / Balance
how would gluing your veins keep your mouth shut smh
gives anyone a reason to even try make mute toxin since its very hard to get + glue is abundant, so now if you want mime gun you cant use glue

## Technical details
no

## Media

https://github.com/space-wizards/space-station-14/assets/39013340/d34bcab8-79cd-41eb-af22-6bb46852ed4f

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by brainfood1183** [comment]
can we use glue to stop bleeding?

**Comment by deltanedas** [comment]
real

**Comment by JoeHammad1844** [comment]
> can we use glue to stop bleeding?

tranexamic acid spear

**Comment by EmoGarbage404** [comment]
if you want this to be extremely based, make it so that glue also mutes when sprayed on someone :trollface:

**Comment by deltanedas** [comment]
touch time

**Comment by JoeHammad1844** [comment]
would be epic if you could use spears to mute people actually, budget mute toxin so your target cant scream while you kill them

**Comment by deltanedas** [comment]
its not budget mute toxin thats literally just mute toxin except spawns in maints regularly

**Comment by JoeHammad1844** [comment]
syndies should be able to buy mute toxin

**Comment by UbaserB** [comment]
> syndies should be able to buy mute toxin

cant you just craft it

**Comment by deltanedas** [comment]
it takes ground uranium among other things and ive never seen it crafted

also in tg i think mute toxin (equivalent?) comes in a poison kit for like 4tc so that could be done

**Comment by JoeHammad1844** [comment]
requires chemistry AND slav to be mining so you are only really gonna get it if you are a chemist and convince cargo

File: Resources/Prototypes/Reagents/cleaning.yml [hunk 1]
```diff
@@ -67,6 +67,19 @@
   viscosity: 0.5
   tileReactions:
     - !type:SpillTileReaction
+  reactiveEffects:
+    Acidic:
```
Comment (Partmedia) [inline Resources/Prototypes/Reagents/cleaning.yml:71]: Why should touching this reagent (or slipping on a puddle of it and falling on it and having it transfer) mute you?

Comment (deltanedas) [reply Resources/Prototypes/Reagents/cleaning.yml:71]: removed

**Pull request merged**